### PR TITLE
RHNConf documentation for 1.x branch

### PR DESCRIPTION
**Pull request by PaulWay**
Adding documentation and some code coverage test improvements for the RHNConf parser.

### CODE REVIEW

**Comment by PaulWay** [comment]
BTW, what's the correct syntax to do a reference in Sphinx to the 'LegacyItemAccess' class definition?

**Comment by xiangce** [comment]
~~~
:class:`insights.core.LegacyItemAccess`
~~~
works

File: insights/parsers/rhn_conf.py [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
 @parser('rhn.conf')
 class RHNConf(LegacyItemAccess, Parser):
-    """Class to parse the configuration file ``rhn.conf``
+    """
+    Class to parse the configuration file ``rhn.conf``.
+
+    The special feature of ``rhn.conf`` is that values can span multiple
+    lines with each intermediate line ending with a comma.
+
+    This parser uses the :py:class::LegacyItemAccess mix-in to provide access to
```
Comment (bfahr) [inline insights/parsers/rhn_conf.py]: You can use the syntax recommended by @xiangce.  Here's the Sphinx reference: http://www.sphinx-doc.org/en/stable/domains.html#cross-referencing-python-objects

Comment (PaulWay) [inline insights/parsers/rhn_conf.py]: OK, great, thanks @xiangce and @bfahr - will use that in the future!  Should be ready to go.

**Pull request merged**